### PR TITLE
Fix identity

### PR DESCRIPTION
**Pull request by willemolding**
Fixes the issue where only one non-test agent could ever register. Also includes the update to 0.0.9

### CODE REVIEW

File: dna-src/zomes/graphql/code/src/identity/mod.rs [hunk 1]
```diff
@@ -31,10 +33,10 @@ pub fn get_identity(agent_id: &Address) -> ZomeApiResult<Identity> {
 }
 
 pub fn register_user(name: String, avatar_url: String, hylo_id: String) -> ZomeApiResult<Address> {
-    // don't reregister everyone if it has been done already
+    // only register the test identities if there are none already
     let n_registered = get_people()?.len();
-    if n_registered > 0 {
```
Comment (willemolding) [inline dna-src/zomes/graphql/code/src/identity/mod.rs:36]: This was the offending line. It aborts with a fake success if it detects existing agents. This explains the behavior we were seeing when testing online where a registration would appear to work but not actually

Comment (lorenjohnson) [reply dna-src/zomes/graphql/code/src/identity/mod.rs:36]: Great news! We can now bring on the n3h magic.

File: container-config.toml [hunk 2]
```diff
@@ -49,3 +47,15 @@ id = "ui-interface"
...
 port = 4000
 dna_interface = "websocket_interface"
+
+# [logger]
+# type="debug"
+# [[logger.rules.rules]]
+# exclude=true
+# pattern=".*"
+
+[network]
```
Comment (lorenjohnson) [inline container-config.toml]: Leave this commented out by default?

File: dna-src/zomes/graphql/code/src/identity/mod.rs (hunk 1 above)
Comment (lorenjohnson) [reply dna-src/zomes/graphql/code/src/identity/mod.rs:36]: Does this mean we now won't have the Beatles at all once there is more than 1 real agent seen? (which is fine/better)

Comment (willemolding) [reply dna-src/zomes/graphql/code/src/identity/mod.rs:36]: kinda. The first agent will add the Beatles and the other agents will get them via gossip

**Review by lorenjohnson — APPROVED** [review]
LGTM 👍  ...and we're off the races. Can't wait to try with 0.0.10 improvements.

**Review by philipbeadle — APPROVED** [review]

**Pull request merged**